distributed-exceptions/README.md

```diff
@@ -0,0 +1,37 @@
+# Distributed exceptions
+In this demo we will how to propagate errors as exceptions in a distributed environment.
+
+One of the most common ways to indicate that a logical error has occurred and that a Command handling failed is to throw an exception from a
+ Command handler. However, if the exception is directly serialized there is no guarantee that the command sending side can properly
+  deserialize the
+ exception in question. 
+ 
+That is why by default Axon Framework will wrap any exception thrown in a Command handler in a `CommandExecutionException`.
+
+There are a couple of different approaches on how communicate logical, business errors, and failures between services and one of them is
+ shown in this example.
+
+These are some points to pay attention to: 
+1. Instead of using generic `IllegalArgumentException` or `IllegalStateException` different failures can be expressed as exception
```

Comment:
```suggestion
1. Instead of using generic `IllegalArgumentException` or `IllegalStateException`, different failures can be expressed as exception
```